### PR TITLE
Update to CE3 and revert fix to reset the IO runtime

### PR DESCRIPTION
**Pull request by cquiroz**
Cats effect 3.2.0 provides a way to reset the js global runtime. We can use this in the context of HMR

### CODE REVIEW

File: explore/src/main/scala/explore/Explore.scala [hunk 1]
```diff
@@ -201,13 +200,14 @@ object ExploreMain {
...
       }
-      .onError { t =>
+      .void
+      .handleErrorWith { t =>
         logger.error(t)("Error initializing") >>
           crash(s"There was an error initializing Explore:<br/>${t.getMessage}")
       }
   }
+
+  // TODO Release dispatcher on stop.
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/Explore.scala:212]: Given the nature of web apps, I think we can get away with not implementing this.

**Review by rpiaggio — APPROVED** [review]
Nice LGTM

File: explore/src/main/scala/explore/Explore.scala (hunk 1 above)
Comment (cquiroz) [reply explore/src/main/scala/explore/Explore.scala:212]: Right, I wasn't sure about that

**Review by toddburnside — APPROVED** [review]
LGTM. 😄

**Pull request merged**